snuba/cli/multistorage_consumer.py

```diff
@@ -213,6 +218,15 @@ def multistorage_consumer(
             consumer_configuration, producer=producer, commit_log_topic=commit_log,
         )
 
+    dead_letter_producer: Optional[KafkaProducer] = None
+    dead_letter_queue: Optional[Topic] = None
+    if dead_letter_topic:
+        dead_letter_queue = Topic(dead_letter_topic)
+
+        dead_letter_producer = KafkaProducer(
```

Comment:
Who is closing the KafkaProducer? All producers in Arraoyo need to be closed (method `close`)  at the end of the script, otherwise the producer will hang during shutdown

Reply:
added here https://github.com/getsentry/snuba/pull/2437/files#diff-fdeca11abbdfdf36e4aa9604b39c17a937f1c35368351d1f20740a129c7b6478R258